docs/copied-from-beats/docs/security/users.asciidoc

```diff
@@ -197,156 +344,245 @@ information.
 |Send monitoring data to the monitoring cluster
 |====
 
-. Assign the following role to users who will view the monitoring data in
-{kib}:
+TIP: See <<monitoring-metricbeat-collection>>
+for complete details on setting up Metricbeat collection.
+--
+
+[float]
+[[privileges-to-publish-monitoring-view]]
+==== View monitoring data
```

Comment:
I don't want to go beyond the scope of this PR, but have you considered creating a dedicated monitoring section in the [Kibana APM app](https://www.elastic.co/guide/en/kibana/7.7/xpack-apm.html) and moving this section there or moving it to [Kibana Stack monitoring](https://www.elastic.co/guide/en/kibana/7.7/xpack-monitoring.html) section and then cross-linking to it? This role has nothing to do with the APM Server and for me it wouldn't be the first place to look for. This might be a larger undertaking, as far as I can see we do not have a privileges/roles section in the Kibana APM app yet, so I'm perfectly fine with keeping it here for now, but maybe consider a follow up change.